database/controllers.js

```diff
@@ -11,4 +12,38 @@ const getAllDocuments = (itemId, callback) => {
   });
 };
 
-module.exports = getAllDocuments;
+
+exports.getDoc = (itemId, callback) => {
+  Item.findOne({ id: itemId }, (err, item) => {
```

Comment:
Purely a comment since it works as-is. I (very) recently realized that I could drop lines 17-22 by simply passing the callback here since the callback has the error handling. Applies to the other methods below as well.

Reply:
I'm going to ask help desk about this tomorrow, I totally see what you're saying, and it makes sense, but I thought there was a reason for only sending back the response or the error. Like perhaps you might be adding to your server load with this pattern? since we're really getting heavy into back end, and are going to be thinking about separate instances for the DB and Server, and even clustering, I think this would be an excellent question for help desk tomorrow. I could be totally wrong.